### ISSUE TITLE
Release latest version to Firefox addons store

### ISSUE THREAD

**Issue by samgomena** [SafetyCulture/grpc-web-devtools#174]
The latest version of this tool released to the [Firefox addons store](https://addons.mozilla.org/en-US/firefox/addon/grpc-web-developer-tools/) is `v1.2.1`, from 2019:

<img width="474" alt="image" src="https://github.com/SafetyCulture/grpc-web-devtools/assets/13566124/91378b41-ce10-4b48-a52c-7aee4545b838">

Can you please upload the latest release to the Firefox addons store?

Thanks!

**Comment by aapeliv** [comment]
Yes, would be great to see the latest changes released for firefox!
